nlp.py

```diff
@@ -18,12 +17,11 @@ def extract_characters(text):
 
     try:
         nlp = spacy.load("en_core_web_md")
-    except:
+    except Exception:
```

Comment:
Can we be more specific about what kind of Exception we should be catching, e.g RuntimeError, or will the linter complain?

Reply:
Done :)